ops_omgx/monitor/transaction-monitor/services/chain-scanner.service.js

```diff
@@ -63,9 +63,14 @@ class ChainScannerService extends DatabaseService {
     // scan the chain and get the latest number of blocks
     this.latestBlock = await this.L2Provider.getBlockNumber();
 
+    // check the latest block on MySQL
+    await this.query(`USE OMGXRinkeby`);
+    let latestSQLBlockQuery = await this.query(`SELECT MAX(CAST(blockNumber AS FLOAT)) from block`);
```

Comment:
The blockNumber is always the integer.
```
SELECT MAX(blockNumber) from block
```
Please move the querying code to `database.service.js`.